### PR TITLE
Adding log measure

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Add doc for measure

### Motivation

It's gonna be released

### Preview link

* https://docs-staging.datadoghq.com/gus/log-mesure/logs/explore/#mesures
* https://docs-staging.datadoghq.com/gus/log-mesure/logs/explore/#log-graph

### CODE REVIEW

**Comment by NBParis** [comment]
@l0k0ms Title issue: the A is missing (Measure).

The picture in the 3. is not centralized

File: content/logs/explore.md [hunk 1]
```diff
@@ -91,14 +91,32 @@ To start using an attribute as a Facet or in the search, click on it and add it
...
-Once this is done, the value of this attribute is stored **for all new logs** and can be used for searches via the [search bar](#searche-bar) or [Facet Panel](#facet-panel).
+Once this is done, the value of this attribute is stored **for all new logs** and can be used in [the search bar](#search-bar), [the Facet Panel](#facet-panel), and in the [Log graph query](/logs/log-graph).
 
 ### Facet Panel
 
 Use facets to easily filters on your logs. The search bar and url automatically reflect your selections.
 
 {{< img src="logs/explore/facet_panel.png" alt="Facet panel" responsive="true" popup="true" style="width:80%;">}}
 
+## Mesures
```
Comment (NBParis) [inline content/logs/explore.md]: Missing an A

File: content/logs/explore.md [hunk 2]
```diff
@@ -189,11 +207,14 @@ Switch between the Log List and the Log Graph mode by clicking on this button:
...
 To start using it:
 
-1. Choose the Measure to graph
-2. Split by [Tag](/getting_started/tagging) or [Facet](#facets) to split your graph over the desired dimension.
-{{< img src="logs/explore/graph/split_by_log_graph.png" alt="split by log graph" responsive="true" popup="true" style="width:80%;">}}
+1. Choose a [Measure](#measure) or [Facet](#facet) to graph. Choosing a Measure let you then choose the aggregation function whereas selecting a Facet display the unique count.
+    {{< img src="logs/explore/graph/choose_measure_facet.png" alt="choose measure facet" responsive="true" popup="true" style="width:50%;">}}
+2. Select the aggregation function for the Measure you want to graph
+    {{< img src="logs/explore/graph/agg_function_log_graph.png" alt="aggregation function for log graph" responsive="true" popup="true" style="width:50%;">}}
+3. Split by [Tag](/getting_started/tagging) or [Facet](#facets) to split your graph over the desired dimension.
```
Comment (NBParis) [inline content/logs/explore.md]: Image is not centralized

File: content/logs/explore.md (hunk 1 above)
Comment (l0k0ms) [reply content/logs/explore.md]: 👌

**Pull request merged**